.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
+  - "8"
```

Comment:
@djfarrelly shall we use then `node` or `lts/*` instead of `8`? https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions

Reply:
@philippemiguet I think just `lts/*` should work which is now node 10 and we should be using only LTS releases for our production services 😄 . If this ends up using language features that only exist in node 10 and we need to polyfill for node 8 or below we can add `8`. I think this repo should use pretty standard/stable language features so running the tests against 10 😄